tests/test_layers_convolution.py

```diff
@@ -61,9 +61,17 @@
 n = tl.layers.DeConv2d(nin, n_filter=32, filter_size=(3, 3), strides=(2, 2), name='DeConv2d')
 print(n)
 shape = n.outputs.get_shape().as_list()
+print(shape[1:])
 # if (shape[1] != 200) or (shape[2] != 200) or (shape[3] != 32): # TODO: why [None None None 32] ?
-if (shape[3] != 32):
+if (shape[1:] != [200, 200, 32]):
```

Comment:
You could have kept the tests that you implemented before. Do you need help on this ?